drv/lpc55-gpio-api/src/lib.rs

```diff
@@ -145,24 +145,64 @@ pub enum Digimode {
     Digital = 1,
 }
 
+impl From<bool> for Digimode {
+    fn from(state: bool) -> Self {
+        if state {
+            Digimode::Digital
+        } else {
+            Digimode::Analog
+        }
+    }
+}
+
```

Comment:
Can you give me a pointer to where these functions actually get used in your code? I'm still not sure where they are actually needed.

Reply:
Bah.. I am not using them. They are leftover from some short-lived debugging.

Reply:
Removed